components/Footer/index.js

```diff
@@ -47,14 +61,26 @@ const NavLink = styled.li`
   align-items: center;
   list-style-type: none;
   padding: 0px 5px;
+  color: ${(props) => (props.active ? "#00ff00" : "#95091b")};
+  cursor: pointer;
+  a {
+    color: #95091b;
+    text-decoration: none;
+    &:hover {
+      color: #099584;
+      font-size: 1.1em;
+      transform: scale(1.1);
+    }
+  }
 `;
 
 const LinkList = styled.ul`
   display: flex;
   flex-direction: row;
```

Comment:
```suggestion
```

Reply:
`flex-direction: row;` almost never needs to be specified since it's the default anyway. :)

Reply:
Thank you. Corrected! :)